### PR TITLE
Jenkins does cleanup on its own, this just raises exceptions

### PR DESCRIPTION
**Pull request by Pandentia**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -22,18 +22,12 @@ pipeline {
     post {
```
Comment (Pandentia) [inline Jenkinsfile]: you have also seem to have forgotten about the `always` flag

Comment (NamedKitten) [inline Jenkinsfile]: Always is before any other post action so changing it wouldn't work.

Comment (NamedKitten) [inline Jenkinsfile]: Also git submodules tend to break if I don't completely delete everything.

**Comment by Pandentia** [comment]
the problem is that you're not deleting everything because Jenkins can't
delete the docker container

if you're that desperate, add a
shell 'rm -rf * | true'

**Comment by NamedKitten** [comment]
Can Jenkins reset the container every time?

**Comment by Pandentia** [comment]
Just use shell 'rm -rf * | true' if you're that paranoid.

**Pull request closed without merging**